app/models/institution.rb

```diff
@@ -233,32 +233,29 @@ def self.autocomplete(search_term, limit = 6)
   scope :search, lambda { |search_term, include_address = false, fuzzy_search = false|
     return if search_term.blank?
 
-    clause = [
-      'facility_code = (:facility_code)',
-      'institution LIKE (:upper_search_term)',
-      'city LIKE (:upper_search_term)'
-    ]
+    clause = ['facility_code = :facility_code']
 
     if fuzzy_search
-      clause << "SIMILARITY(institution, :search_term) > #{Settings.institution_name_similarity_threshold}"
-      clause << "SIMILARITY(city, :search_term) > #{Settings.institution_city_similarity_threshold}"
-      clause << 'zip = (:search_term)'
-      clause << 'ialias LIKE (:upper_search_term)'
+      clause << 'institution % :upper_search_term'
```

Comment:
I see that the SIMILARITY and LIKE clauses for city and institution were replaced with the % wildcard character. Are the similarity thresholds no longer needed?

Reply:
`SIMILARITY` compares the entire search term with the column value, instead of fully leveraging `pg_trgm`. We'll be adding the similarity thresholds back in, but outside of that clause so that we're only enforcing it on the subset of institutions that already match, instead of checking the similarity of ALL of them (which was causing the search results to exceed the staging timeout values)